controllers/awsmachine_controller_test.go

```diff
@@ -60,9 +61,16 @@ var _ = Describe("AWSMachineReconciler", func() {
 		klog.SetOutput(GinkgoWriter)
 		var err error
 
+		awsmachine := &infrav1.AWSMachine{
+			ObjectMeta: metav1.ObjectMeta{
+				Name: "",
```

Comment:
Why empty name?

Reply:
no good reason ... I think I saw another example test that used this. Updated it to "test"